test/blackbox-tests/test-cases/dune-project-meta/basic-generate.t

```diff
@@ -29,12 +29,28 @@ The `dune build` should generate the opam file
   > (package
   >   (name cohttp-async)
   >   (synopsis "HTTP client and server for the Async library")
-  >   (description "A _really_ long description")
+  >   (description "\
+  > A _really_long multi line description that spans across lines to
+  > make sure that the rendering of long strings stays compatible.
+  > ")
   >   (depends
   >     (cohttp (>= 1.0.2))
   >     (conduit-async (>= 1.0.3))
   >     (async (>= v0.10.0))
   >     (async (< v0.12))))
+  > ;
+  > (package
+  >   (name cohttp-lwt)
+  >   (synopsis "HTTP client and server for the Lwt library")
+  >   (description "
```

Comment:
is it intended that the first character of the description is a newline?

Reply:
Yes. This is taken from from the regression in dune generating it's own opam files that Rudi reported, where the description in `dune.opam` used to start with an empty line and doesn't anymore.

I think the new way is sort of nicer but we probably don't want all OPAM files to be reformatted as a consequence of us bumping the version of our vendored o-f-f.

Reply:
ok. can you update the string explaining that? (or at least change the "multi line" part to "a string that starts with a newline")

Reply:
Done; I hope it is clearer now.